contracts/SignerBondsUniswapV2.sol

```diff
@@ -75,13 +76,22 @@ contract SignerBondsUniswapV2 is ISignerBondsSwapStrategy, Ownable {
     // Determines the deadline in which the swap transaction has to be mined.
     // If that deadline is exceeded, transaction will be reverted.
     uint256 public swapDeadline = 20 minutes;
+    // Determines if the open auctions check should be performed. If true,
+    // swaps cannot be performed if open auctions count is bigger than zero.
+    // If false, open auctions are not taken into account.
+    bool public openAuctionsCheck = true;
 
     event UniswapV2SwapExecuted(uint256[] amounts);
 
-    constructor(IUniswapV2Router _uniswapRouter, CoveragePool _coveragePool) {
+    constructor(
+        IUniswapV2Router _uniswapRouter,
+        CoveragePool _coveragePool,
```

Comment:
We can also pass `IERC20 _collateralToken, address _recipient` to reduce problems with dependencies.

Reply:
I think we could leave as is due to https://github.com/keep-network/coverage-pools/pull/88/commits/3f36835113dac64bdd05101fd10fd69b0c143f3a